CHANGELOG.md

```diff
@@ -37,6 +37,57 @@ CDKTF is bumping the minimum supported version of Terraform from 1.0 to 1.2 star
+- chore: create TFC/TFE workspace using the Terraform CLI [\#2424](https://github.com/hashicorp/terraform-cdk/pull/2424)
+- chore: switch to SHA-based workflow pins [\#2422](https://github.com/hashicorp/terraform-cdk/pull/2422)
+- chore: Update all documentation examples for Typescript [\#2421](https://github.com/hashicorp/terraform-cdk/pull/2421)
+- chore: remove long-running hooks [\#2418](https://github.com/hashicorp/terraform-cdk/pull/2418)
+- chore: translate documentation to csharp [\#2417](https://github.com/hashicorp/terraform-cdk/pull/2417)
+- chore: add a refactoring guide [\#2413](https://github.com/hashicorp/terraform-cdk/pull/2413)
+- chore: fix homebrew deploy step [\#2412](https://github.com/hashicorp/terraform-cdk/pull/2412)
+- chore: add TFE information to TFC docs [\#2407](https://github.com/hashicorp/terraform-cdk/pull/2407)
+
+### fix
+
+- fix(cli): don't enquote args as they will be escaped and end up in the calls to Terraform [\#2493](https://github.com/hashicorp/terraform-cdk/pull/2493)
+- fix: detect destroys on windows / TF version [\#2486](https://github.com/hashicorp/terraform-cdk/pull/2486)
+- fix(tests): change Options -> Config as per #2471 [\#2482](https://github.com/hashicorp/terraform-cdk/pull/2482)
+- fix(cli): windows fixes [\#2480](https://github.com/hashicorp/terraform-cdk/pull/2480)
+- fix: limit concurrency for integration tests to 10 for each windows and linux [\#2474](https://github.com/hashicorp/terraform-cdk/pull/2474)
+- fix(tests): Make version more stable [\#2467](https://github.com/hashicorp/terraform-cdk/pull/2467)
+- fix: use older SHA from jsii superchain docker image [\#2465](https://github.com/hashicorp/terraform-cdk/pull/2465)
+- fix: fix go build in pipeline [\#2464](https://github.com/hashicorp/terraform-cdk/pull/2464)
+- fix(cli): remove newlines from tokens during terraform login [\#2459](https://github.com/hashicorp/terraform-cdk/pull/2459)
+- fix(cli): abort apply if not all variables are known [\#2455](https://github.com/hashicorp/terraform-cdk/pull/2455)
+- fix: Expose provider matcher to jest [\#2448](https://github.com/hashicorp/terraform-cdk/pull/2448)
+- fix: Adds `environment` to keys to preserve case [\#2446](https://github.com/hashicorp/terraform-cdk/pull/2446)
+- fix: Allow git access for sentry cli in docker [\#2426](https://github.com/hashicorp/terraform-cdk/pull/2426)
+- fix: add path: dist for npm and pypi [\#2410](https://github.com/hashicorp/terraform-cdk/pull/2410)
+
+### feat
+
+- feat: use our own pre-built package for node-pty to have builds for Nodejs 19 available [\#2478](https://github.com/hashicorp/terraform-cdk/pull/2478)
+- feat(cli): support -var and -var-file terraform flags [\#2468](https://github.com/hashicorp/terraform-cdk/pull/2468)
+- feat(cli): add -no-color tag [\#2307](https://github.com/hashicorp/terraform-cdk/pull/2307)
+- feat(cli): Add cdktf provider list command [\#2270](https://github.com/hashicorp/terraform-cdk/pull/2270)
```

Comment:
only just noticed that we don't even mention this feature in the blog post 😅
Maybe we can move the this feature section at the top of the changelog and move the list command PR to the top of this list as some kind of "best effort visibility fix"? 😄

Reply:
Yeah I left it out because it felt detached to the other work we have been doing and I wanted to paint a cohesive picture of this releases efforts. I'm open to add a small section on top

Reply:
I didn't see you comment and just reordered the sections as I proposed 😅
I'm okay with not promoting it in the changelog, as we mostly focus on breaking changes in sections we put in there.